content/blog/webrtc-demo.md

```diff
@@ -0,0 +1,192 @@
+* Totally redesign the application to improve user experience
+* Set up phone & video call, transfer & indirect transfer
+* Being able to call another number while already on call & switch between interlocutors
+
+## Launching the demo at home
+
+If you want to take a closer look at the code and the demo itself, you can go and set it up with these commands: 
+```
+apt install wazo-plugin-cli
+wazo-plugind-cli -c 'install git https://github.com/wazo-platform/wazo-webrtc-demo'
+```
+Otherwise, you can clone the following repository : [wazo-webrtc-demo](https://github.com/wazo-platform/wazo-webrtc-demo) and follow the instructions in the README file on the master branch. 
+
+Feel free to report any bugs, I will occasionally take a look.  
+
+
+## Redesigning the interface
+
+I prepared some screenshots so you can notice the difference between the application before and after I redesigned it. The main styling was based on a few mock ups. You can take a look here at the home page mock up. 
+
+![Mockup](/images/blog/webrtc-demo/mockup.png "Mock up")
+
+What you can’t see in the screenshots is the way the interface is moving. I decided to animate every button : when you hover them, they fill up. The same goes for the logout button : when you hover it, it spins backwards and the real logout button appears. I also looked up on the internet to find better backgrounds : a landscape at dawn for the login page, and a landscape at daylight for the rest of the application. These images are royalty-free and created by Jcomp (source: Freepik).
+
+Now let’s take a look at those screenshots !
+
+![Login&Home](/images/blog/webrtc-demo/login&home.png)
+![In&OnCall](/images/blog/webrtc-demo/in&oncall.png)
+![Video&MultiCalls](/images/blog/webrtc-demo/video&multi.png)
+
+In the last screenshot, you can see the multiple calls display styling. I would like to dig deeper into this feature, and take a look at the code behind it. 
+
+## Zoom on the multiple calls display
+
+
+First, let’s go into the `dialer.js` file. This file contains every function that will build our dialer (from dialing to video calling and receiving calls). On top of this file, we create an empty object : `sessions`, which is a Wazo object you can use to store your call sessions. On the other hand, `currentSession` is initialized and will be used later. 
+
+```js
+  const sessions = {};
+  let currentSession;
+```
+
+Now let’s study the `initializeWebRtc` function. In the Wazo SDK you have a bunch of useful functions to create a Web RTC application. You set up all of the events and write the names of the functions you’ll be using for these interactions (from line 9 to 17).
+
+But the interesting part for the multiple calls handler is in the `Wazo.Phone.ON_REINVITE` event. We set up the `callSession` variable on the corresponding session id (we’re looking for it into our sessions object).
```

Comment:
The update of the `realDisplayName` is useful for transfer, not multiple call

Reply:
I may have misinterpreted, I'm correcting that right away